### PR TITLE
feat: Testing alpha releases of JS libs

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

We need to test the posthog-ts refactor as well as the new js-lite libs for Node and Web( used by the toolbar). This PR includes all of them!

## Changes

* Adds the new libs with some small tweaks to their new APIs

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

- Checked locally that the toolbar, web and plugin-server events happen
- Checked that on the local demo page the toolbar sends events to app.posthog.com

### CODE REVIEW

File: plugin-server/src/worker/ingestion/team-manager.ts [hunk 1]
```diff
@@ -215,13 +215,16 @@ DO UPDATE SET property_type=$5 WHERE posthog_propertydefinition.property_type IS
...
             const distinctIds: { distinct_id: string }[] = organizationMembers.rows
             for (const { distinct_id } of distinctIds) {
-                posthog.identify(distinct_id)
-                posthog.capture('first team event ingested', {
-                    team: team.uuid,
-                    sdk: properties.$lib,
-                    realm: properties.realm,
-                    host: properties.$host,
-                    $groups: {
+                posthog.capture({
```
Comment (pauldambra) [inline plugin-server/src/worker/ingestion/team-manager.ts:218]: no need to identify?

**Review by pauldambra — COMMENTED** [review]
Looks good code wise... (I've not run it :))

It's a lot to change all at once. If things break will we know the source of the problem?

File: plugin-server/src/worker/ingestion/team-manager.ts (hunk 1 above)
Comment (benjackwhite) [reply plugin-server/src/worker/ingestion/team-manager.ts:218]: I don't think so no. We aren't saving any properties against a user and the distinctId is the distinctId :D 

Identify never made sense for the way it was implemented before as it was caching the distinctId for all subsequent calls :o and doing aliasing and all sorts of bad stuff. As we are capturing a fresh event

**Comment by benjackwhite** [comment]
> Looks good code wise... (I've not run it :))
> 
> It's a lot to change all at once. If things break will we know the source of the problem?

Not worried too much about the changes as:
1. Web is just swapping to the typescript one (which we did before and it seemed good)
2. Toolbar is changing to the js-lite so _worst_ case there we lose some toolbar tracking events
3. Node is swapping (bigger change) but I'm pretty sure its in a bad state atm anyways due to the identify aliasing bug behaviour. 
4. I'm gonna keep a keen eye on each of those things for the first hour after release ;)

File: plugin-server/src/worker/ingestion/team-manager.ts (hunk 1 above)
Comment (benjackwhite) [reply plugin-server/src/worker/ingestion/team-manager.ts:218]: The old js-lite was working as a client lib here when it should have been behaving like a server-side lib

**Pull request merged**